service-lib/src/test/kotlin/fi/espoo/voltti/logging/logback/AppenderTest.kt

```diff
@@ -80,7 +80,7 @@ class AppenderTest {
             it.assertSanitized().containsExactly(event1.tuple())
             it.withLatestSanitized { actual ->
                 defaultErrorAssertions(actual, meta, exception)
-                assertThat(actual["stackTrace"] as String).doesNotContain(testSSN)
+                assertThat(actual["stackTrace"] as String).doesNotContain(UUIDWithSSN)
```

Comment:
Should assert that `UUIDWithSSN` _is_ in the log message.

Reply:
That should've asserted `message` instead of `stackTrace` since the UUID is there 😢 
Fixed the `stackTrace` assertions and added a couple for `message`